### PR TITLE
get rid of dependency fs-extra

### PR DESCRIPTION
**Pull request by dotlambda**
Feel free to close if you think this is not worth it.

### CODE REVIEW

**Comment by Simon-Laux** [comment]
not possible with our current target of node 12, also I personally like using `fs-extra`, still might be something we could do after upgrading our node target version.

**Comment by dotlambda** [comment]
> current target of node 12

Then you should not use `@types/node` v14.

But this was also done in https://github.com/deltachat/deltachat-node/pull/514, so that would have to be reverted if we stick with Node.js 12.

> I personally like using `fs-extra`

I don't know it very well, but felt like the additional functions it provides don't always have well-defined semantics.

File: bin/test_package_lock_version.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const { readJson } = require('fs-extra')
+const { readFile } = require('fs-extra')
```
Comment (Simon-Laux) [inline bin/test_package_lock_version.js:1]: still one reference here

Comment (Simon-Laux) [reply bin/test_package_lock_version.js:1]: I'll merge and fix on the other branch

**Pull request merged**